src/renderer/chunk_meshing.zig

```diff
@@ -1210,13 +1210,21 @@ pub const ChunkMesh = struct { // MARK: ChunkMesh
 		const oldBlock = self.chunk.data.getValue(chunk.getIndex(x, y, z));
 
 		if(oldBlock == newBlock) {
+			if(newBlock.blockEntity()) |blockEntity| {
```

Comment:
Same code and same **error message** twice, you ment to create a function right?